doc/source/getting_started/intro_tutorials/08_combine_dataframes.rst

```diff
@@ -155,8 +155,10 @@ index. For example:
 
 .. ipython:: python
 
-    air_quality_ = pd.concat([air_quality_pm25, air_quality_no2],
-                             keys=["PM25", "NO2"])
+    air_quality_ = pd.concat([
+        air_quality_pm25, air_quality_no2],
+        keys=["PM25", "NO2"
+    ])
```

Comment:
I think the brackets should be on their own lines unless this can fit on one line